lib/compliance/parsepdf417-us.js

```diff
@@ -97,7 +97,14 @@ function normalize (result, version, issuingAuthorityNumber) {
 
   if (result.country === 'USA' && version > 1) {
     result.dateOfBirth = date.substr(4, 4) + date.substr(0, 4)
+    result.expirationDay = expirationDate ? expirationDate.substr(2, 2) : '00'
+    result.expirationMonth = expirationDate ? expirationDate.substr(0, 2) : '00'
+    result.expirationYear = expirationDate ? expirationDate.substr(4, 4) : '00'
```

Comment:
Can't we just use the current expirationDate to perform the test? We don't really need to create the additional fields.